Editor/PackageManager/PackageOperationsManager.cs

```diff
@@ -150,13 +206,43 @@ private static void WaitForPackageStatus()
             else
             {
                 packageRequestStatus = PackageRequestStatus.None;
+                OnPackageEnabled?.Invoke(null, new PackageEnabledEventArgs(addRequest.Result));
```

Comment:
calling events with null feels super odd, tbh

Reply:
🤔

Reply:
In case of static classes, sender should be null.